test/cmdlineTests/yul_optimize_runs/output

```diff
@@ -5,12 +5,12 @@ Pretty printed source:
 object "RunsTest1" {
     code {
         {
-            let _1 := datasize("Runtime")
-            datacopy(0, dataoffset("Runtime"), _1)
+            let _1 := datasize("Runtime_deployed")
+            datacopy(0, dataoffset("Runtime_deployed"), _1)
             return(0, _1)
         }
     }
-    object "Runtime" {
+    object "Runtime_deployed" {
```

Comment:
This case now depends on the name of the subobject... which is not really nice...